Cargo.toml

```diff
@@ -25,8 +25,8 @@ std = []
 rustc-dep-of-std = ['core', 'compiler_builtins']
 
 [dev-dependencies]
-criterion = "=0.3.2" # 0.3.3 broken on Rust 1.33
-humansize = "1.1"
+criterion = "=0.3.2" # benchmark only
```

Comment:
I'd add a short comment explaining that these `# benchmark only` lines sometimes get removed on CI and why.